frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -60,6 +61,20 @@ export class SmartScalar extends Component {
   static noHeader = true;
 
   static settings = {
+    ...fieldSetting("scalar.field", {
```

Comment:
Wdyt about renaming it to "scalar.metric"? It would be aligned with existing names of settings of other charts such as "graph.metrics", "pie.metric", and "funnel.metric"

Reply:
I think it would make sense, but `scalar.field` is already used by Scalar, Gauge, and Progress.  Should we rename it there too?

Reply:
Oh, if it is used already in different charts as well let's keep it